patches/minecraft/net/minecraft/client/Minecraft.java.patch

```diff
@@ -185,7 +185,13 @@
        }
  
        if (this.f_91056_ != null) {
-@@ -1136,6 +_,7 @@
+@@ -1131,11 +_,12 @@
+       int i = this.f_90990_.m_85385_(this.f_91066_.f_92072_, this.m_91390_());
+       this.f_90990_.m_85378_((double)i);
+       if (this.f_91080_ != null) {
+-         this.f_91080_.m_6574_(this, this.f_90990_.m_85445_(), this.f_90990_.m_85446_());
```

Comment:
You should leave this in. The guiLayers list in ForgeHooksClient contains only the added layers, so by removing this call you'd be causing the top layer to not have resize called on it.

Reply:
Fixed